### ISSUE TITLE
[logs] queries to be supported in data analysis

### ISSUE THREAD

**Issue by StefanoCecere** [vgwb/Antura_arabic#168]
What follows is a list of required logged data (and whether it is missing).
Note: there is some confusion over whether 'play sessions' means App Session or Play Session.

Note: for _group_ analysis, all player data should be moved to another DB.

 **1. Curve of Duration of play sessions (one player, a group)**
If this refers to a Play Session: We query _LogJourneyScoreData.TotalPlayTime_ by AppSession.
If this refers to an App Session: We need to query LogInfoData and retrieve play time by comparing timestamps.
**(requires PlayTime logging #182 )**

 **2. Curve of Duration of play per day (one player, a group)**
Same as above, but we filter by timestamps to retrieve the day.

 **3. Nb of mini-game/assessment per play session (one player, a group)**
I guess this referring to an 'app session'.
In that case: We query _LogMinigameScoreData_ filtering by _AppSession_

 **4. Nb of mini-game/assessment per day (one player, a group)**
Same as above, but we filter by timestamps to retrieve the day.

 **5. Learning blocks ID or T0+x date of drop out (one player, a group)**
We may query _LogJourneyScoreData_ to retrieve the progression in play sessions and compare timestamps.

 **6. Antura Space use duration per play session and per day (one player, a group)**
We query LogInfo looking for AnturaSpace (enter) and AnturaSpace (exit)

 **7. Score evolution per mini game (the rough data that we use to create the 3 stars score) (one player, a group)**
We query _LogMinigameScoreData.Score_ and filter by Minigame code.

 **8. The average evolution of score, all minigame (in stars) (one player, a group)**
We query _LogMinigameScoreData.Score_ and filter by Minigame code.

 **9. Evaluate correlation between dropout/non play and score variation/new game apparition (a group)**
We can detect dropout by analyzing LogInfo timestamps and either _JourneyScoreData_ or _LogJourneyScoreData_.
We can detect score variation with _LogMinigameScoreData_.
We can detect new game apparition with _LogMinigameScoreData_ or _MinigameScoreData_

 **10. Average assessment tries per learning block (to identify potential more difficult learning blocks) (one player, a group)**
We can detect failed assessments by querying _LogMinigameScoreData_, filter by assessment codes, and getting the score.

 **11. Ranking of words associated to failure in mini game (one player, a group)**
We can retrieve _LogLearnData_ to check scores, play sessions, and minigames.

 **12. Mod indicator curve (one player, a group)**
We can use _LogMoodData_

 **13. Replayed learning blocks (one player, a group)**
We can use _LogJourneyScoreData_ to detect multiple plays for the same play sessions (and thus learning blocks).

 **14. More used outfits, colors and accessories, on Antura (one player, a group)**
We need to create a LogInfo event that tracks a change in customization

To recap, what is missing is:

- [x] LogInfo event for customization change (with IDs of customizations)
- [x] PlayTime logging
